rpc/src/tests/ws.rs

```diff
@@ -34,10 +34,9 @@ pub fn serve() -> (Server<ws::Server>, usize, GuardedAuthCodes) {
 	let authcodes = GuardedAuthCodes::new();
 	let stats = Arc::new(informant::RpcStats::default());
 
-	let res = Server::new(|remote| ::start_ws(
+	let res = Server::new(|_remote| ::start_ws(
```

Comment:
Why not just `Server::new(|_| …`?

Reply:
No reason really. I like to label those unused variables just to make it easy to know what's there but I'm happy to change it.